apps/someone-else/controllers/arrange.js

```diff
@@ -32,24 +32,7 @@ ArrangeController.prototype.processDate = function processDate(key, values) {
 };
 
 ArrangeController.prototype.process = function process(req) {
-  if (req.form.values['arrange-collection-radio'] === 'someone-else') {
-    delete req.form.values['no-reason'];
-    req.form.values.nominating = 'Nominate';
-    this.options.next = '/reason';
-  }
-
-  if (req.form.values['arrange-collection-radio'] === 'change-person') {
-    req.form.values['no-reason'] = true;
-    req.form.values.nominating = 'Change';
-    this.options.next = '/personal-details-no-reason';
-  }
-
-  if (req.form.values['arrange-collection-radio'] === 'cancel-request') {
-    this.options.next = '/exit-cancel-request';
-  }
-
```

Comment:
I wanted the buttery biscuit controller but instead I have just a dry water biscuit